db/migrate/20211119104835_add_property_info_fields.rb

```diff
@@ -0,0 +1,12 @@
+class AddPropertyInfoFields < ActiveRecord::Migration[6.1]
+  def change
+    change_table :case_logs, bulk: true do |t|
+      t.column :do_you_know_the_postcode, :integer
```

Comment:
Am I right in thinking these fields don't need adding to the API docs because they're UI only and we're not expecting API clients to pass them? If so they might need to be added to the optional fields in the CaseLog model so that submission validation doesn't fail when they're not provided?